src/vs/workbench/contrib/chat/common/chatModel.ts

```diff
@@ -980,6 +980,7 @@ export interface ISerializableChatData2 extends ISerializableChatData1 {
 export interface ISerializableChatData3 extends Omit<ISerializableChatData2, 'version' | 'computedTitle'> {
 	version: 3;
 	customTitle: string | undefined;
+	isToolsAgentModeEnabled?: boolean;
```

Comment:
Do you still need this? It looks like you're only reading it off the transferred data now

Reply:
Thanks! removed this and updated IChatTransfer.